### PR TITLE
manage the kafka for balrog cluster via operate-first

### PR DESCRIPTION
**Pull request by harshad16**
manage the kafka for balrog cluster via operate-first
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

- Reduced the resource for opf-kafka, as it not being to its capacity as of now.
- set all the manifest for kafka w.r.t balrog setup

### CODE REVIEW

File: kafka/overlays/balrog/topics/balrog-cluster-audit-logs.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaTopic
+metadata:
+  name: balrog-prod.cluster-logs.audit
```
Comment (4n4nd) [inline kafka/overlays/balrog/topics/balrog-cluster-audit-logs.yaml:4]: I don't think the cluster logging topics are needed in the balrog cluster

File: kafka/overlays/balrog/opf-kafdrop.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            - name: secrets
+              mountPath: /mnt/secrets
+              readOnly: true
+            - name: keystore
+              mountPath: /mnt/keystore
+              readOnly: true
+      volumes:
+        - name: secrets
+          secret:
+            secretName: opf-kafka-cluster-ca-cert
```
Comment (4n4nd) [inline kafka/overlays/balrog/opf-kafdrop.yaml:68]: the kafka cluster name and secrets names might have changed

**Review by 4n4nd — CHANGES_REQUESTED** [review]

**Comment by harshad16** [comment]
Thanks for the review, made all the change
Ready for review again

File: kafka/overlays/balrog/opf-kafdrop.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            - name: secrets
+              mountPath: /mnt/secrets
+              readOnly: true
+            - name: keystore
+              mountPath: /mnt/keystore
+              readOnly: true
+      volumes:
+        - name: secrets
+          secret:
+            secretName: odh-message-bus-cluster-ca-cert
```
[the remark this answers is not in the document]
Comment (harshad16) [reply kafka/overlays/balrog/opf-kafdrop.yaml:68]: the secrets are created via the kafka

**Comment by HumairAK** [comment]
@4n4nd can you approve if you are good with the changes

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**